res/locales/fr_FR.po

```diff
@@ -347,73 +347,48 @@ msgstr ""
 "    – Une face plane et un point (distance minimale)\n"
 "    – Un cercle ou un arc (diamètre)\n"
 
-#: constraint.cpp:291
+#: constraint.cpp:303
 msgid ""
 "Bad selection for on point / curve / plane constraint. This constraint can "
 "apply to:\n"
 "\n"
-"    * two points (points coincident)\n"
+"    * two or more points (points coincident)\n"
 "    * a point and a workplane (point in plane)\n"
 "    * a point and a line segment (point on line)\n"
 "    * a point and a circle or arc (point on curve)\n"
-"    * a point and a plane face (point on face)\n"
+"    * a point and one to three plane faces (point on face(s))\n"
 msgstr ""
-"Mauvaise sélection pour la contrainte point / courbe / plan. Cette "
-"contrainte peut s'appliquer à :\n"
-"\n"
-"    – Deux points (points coïncidents)\n"
-"    – Un point et un plan de travail (point dans le plan)\n"
-"    – Un point et un segment de ligne (point en ligne)\n"
-"    – Un point et un cercle ou un arc (point sur courbe)\n"
-"    – Un point et une face plane (point sur une face)\n"
 
-#: constraint.cpp:353
+#: constraint.cpp:364
 msgid ""
 "Bad selection for equal length / radius constraint. This constraint can "
 "apply to:\n"
 "\n"
-"    * two line segments (equal length)\n"
+"    * two or more line segments (equal length)\n"
 "    * two line segments and two points (equal point-line distances)\n"
 "    * a line segment and two points (equal point-line distances)\n"
 "    * a line segment, and a point and line segment (point-line distance "
 "equals length)\n"
-"    * four line segments or normals (equal angle between A,B and C,D)\n"
-"    * three line segments or normals (equal angle between A,B and B,C)\n"
-"    * two circles or arcs (equal radius)\n"
+"    * two or more circles or arcs (equal radius)\n"
 "    * a line segment and an arc (line segment length equals arc length)\n"
 msgstr ""
-"Mauvaise sélection pour une contrainte de longueur / rayon égale. Cette "
-"contrainte peut s'appliquer à :\n"
-"\n"
-"    – Deux segments de ligne (longueur égale)\n"
```

Comment:
`Deux ou trois segments de ligne (longueur égale)`

Reply:
@progval This should be two or more, not two or three (as I think you translated it in French).
I'll use:
> au moins deux segments de ligne(s) (longueur égale)

borrowing from the next one. Let me know if it is not appropriate.